src/toil/jobStores/abstractJobStore.py

```diff
@@ -1152,4 +1160,4 @@ def count(l):
 
             # Do the download
             shutil.copyfileobj(readable, counter)
-            return size[0]
+            return size[0], False
```

Comment:
Should we check the permissions of the file copied with `shutil.copyfileobj` or is this always guaranteed to be `False`?

Reply:
@adamnovak also mentioned that it isn't possible to read an executable bit over HTTP or HTTPS and doing it over FTP would involve tweaking other commands so it would need to be a separate issue.

Reply:
`copyfileobj` just ships bytes between file *objects*; it doesn't know anything about permissions.